.github/workflows/pr-differences-mutants.yml

```diff
@@ -112,28 +123,100 @@ jobs:
         uses: stacks-network/actions/stacks-core/mutation-testing/pr-differences@main
         with:
           shard: ${{ matrix.shard }}
-          package-dimension: "big"
+          package: 'stackslib'
+
+  # Mutation testing - Execute on PR on stacks-node package (normal run, no shards)
+  pr-differences-mutants-stacks-node-normal:
+    name: Mutation Testing - Normal, Stacks Node
+
+    needs: check-big-packages-and-shards
+
+    if: |
+      needs.check-big-packages-and-shards.outputs.run_stacks_node == 'true' && 
+      needs.check-big-packages-and-shards.outputs.stacks_node_with_shards == 'false'
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Run Run mutants on diffs
+        env:
+          BITCOIND_TEST: 1
+          RUST_BACKTRACE: full
+        uses: stacks-network/actions/stacks-core/mutation-testing/pr-differences@main
+        with:
+          package: 'stacks-node'
+
+  # Mutation testing - Execute on PR on stacks-node package (run with strategy matrix shards)
+  pr-differences-mutants-stacks-node-shards:
+    name: Mutation Testing - Shards, Stacks Node
+
+    needs: check-big-packages-and-shards
+
+    if: |
+      needs.check-big-packages-and-shards.outputs.run_stacks_node == 'true' && 
+      needs.check-big-packages-and-shards.outputs.stacks_node_with_shards == 'true'
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        shard: [0, 1, 2, 3]
+
+    steps:
+      - name: Run mutants on diffs
+        env:
+          BITCOIND_TEST: 1
+          RUST_BACKTRACE: full
+        uses: stacks-network/actions/stacks-core/mutation-testing/pr-differences@main
+        with:
+          shard: ${{ matrix.shard }}
+          package: 'stacks-node'
+
+  # Mutation testing - Execute on PR on stacks-signer package (normal run, no shards)
+  pr-differences-mutants-stacks-signer-normal:
+    name: Mutation Testing - Normal, Stacks Signer
+
+    needs: check-big-packages-and-shards
+
+    if: |
+      needs.check-big-packages-and-shards.outputs.run_stacks_signer == 'true'
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Run Run mutants on diffs
+        uses: stacks-network/actions/stacks-core/mutation-testing/pr-differences@main
+        with:
+          package: 'stacks-signer'
 
   # Output the mutants and fail the workflow if there are missed/timeout/unviable mutants
   output-mutants:
     name: Output Mutants
 
     runs-on: ubuntu-latest
 
+    if: always()
```

Comment:
Unnecessary?

```suggestion
```

Reply:
This is necessary to ensure that the output job continues to run even if a preceding job fails. This approach allows the output to clearly show the combined end result from the execution of the mutant jobs.